### DISCUSSION TITLE
xtractl install guide?

### DISCUSSION THREAD

**Discussion by bgrotan** [ldproxy/ldproxy#856] [Setup]
**Xtractl** [1] doesn't have a seperate Discussion-section here on Github, so asking here (since it's related to ldproxy)

Tried reading entities and reloading services using xtractl [1], but running "_go get github.com/interactive-instruments/xtraplatform-cli/tree/main/xtractl_"  does not result in creation of _~/go/bin/xtractl_

Is there something missing, or am I doing it wrong?
Running _xtractl_ from the source folder works though (_go run main.go <command> <options>_) 

[1] https://github.com/interactive-instruments/xtraplatform-cli

**Comment by azahnen** [comment]
The idea is to just download the latest [release](https://github.com/interactive-instruments/xtraplatform-cli/releases) und unpack it wherever you want. 

If you want to use it on Windows, I guess we should add a download for that in a future release. Even if Windows is not a supported platform for ldproxy, nothing speaks against using the CLI there.

**Reply by bgrotan** [reply]
No, never Windows. Thanks for the hint.
